src/jsonschema_wtforms/field.py

```diff
@@ -71,7 +71,7 @@ def extract(cls, params: dict, available: set):
                 if 'contentMediaType' in available:
                     ctype = params['contentMediaType']
                     if isinstance(ctype, (list, tuple, set)):
-                        ctype = ','.join(ctype)
+                        ctype = '|'.join(ctype)
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept advocates for comma separated values. What did motivate this change ?

Reply:
To pass this test: https://github.com/novareto/jsonschema_wtforms/blob/0ff085d9594814ea33fe0e2a23cf4966c9ed1851/tests/fields/test_array.py#L127